src/database.ts

```diff
@@ -97,6 +101,7 @@ export const Database = {
         NotificationsSchema,
         NotificationsValidator
     ),
+    AUTH_CODES: initializeModel("auth_codes", SponsorSchema, SponsorValidator),
```

Comment:
change to AuthCodesSchema, AuthCodesValidator

Reply:
do you mean SponsorAuthSchema and SponsorAuthValidator?

Reply:
yeah